openshift-oauth-proxy/values.yaml

```diff
@@ -0,0 +1,27 @@
+---
+image:
+  repository: docker.io/openshift/oauth-proxy
+  tag: v1.1.0
+  pullPolicy: IfNotPresent
+serviceaccount:
+  name: openshift-oauth-proxy
+
+extraArgs:
+- openshift-delegate-urls: '{"/": {"resource": "namespaces", "verb": "get"}}'
```

Comment:
Is this a sensible default? I don't really know what it does. Maybe it's better to comment it out, to show how an extra arg could be provided?

Reply:
Commented it out and fixed some other things. Test was OK.